Add pricing section (HTML + CSS)

Vegamond opened the pull request:
Додано секцію з тарифами: три картки з цінами, адаптив, базовий дизайн.
Розміщено у partials/pricing.html + стилі в css/pricing.css.

kuroimera reviewed (changes requested):
- Прибрату шапку (плюс там не той шрифт прописаний приберете і все стане на місце) . Файл має починатися з тегу section
- додати у секцію ще один дів, який огортатиме весь контент, дати клас контейнер (щоб підключились стилі контейнеру з файлу стилів контейнера)
- додати до section через пробіл і ще клас section, щоб підключалися стилі (в базових стилях прописаний вже)
- прибрати клас pricing-section і відредагувати усі цсс правила які залежали від цієї назви, наприклад pricing-section h2. Краще прописати окремо заголовку його клас, щоб був незалежним і не довелось потім перероблювати. Інших елементів це теж стосується.
- прибрати базові стилі з вашого файлу, вони всі прописані мною вже в окремому файлі —- 
body {
  margin: 0;
  padding: 0;
  font-family: 'Segoe UI', Tahoma, sans-serif;
  background-color: #EEF4F7;
  color: #1A1A1A;
}

+ тут багато неправильно. колір тексту базовий - #0D0B0A, колір бекграунду -#F8FBFE, прикріплю фото де можна подивитись колір
- поміняйте рем на пікселі, багато відступів і розмірів не співпадає плюс команда вже робить у пікселях.
- ознайомтеся детальніше з моїм файлом базових правил, там на параграфи є стилі, на заголовки ні, тому треба детально прописувати font weight, line-height і типу того. Бачу наприклад, що головний заголовок не прописаний, зажирний.
- видаліть всі кольори, які ви ставили на текст. Десь він у вас сірий, десь темно сірий, а на макеті він усюди один (видаліть їх і базовий правильний колір сам стане, він є у базових стилях)
- колір кнопок - C2D9FC, не DDEBFF, на ховері стає C2D9FC с прозорістю 50%(у вас навпаки нормал світлий а ховер темний) (я там сама у ховерах заплуталася, теж буду у себе виправляти)
-прибрати тіні з карток
- до кнопки джоін додати type="button"
-ну загалом я вже казала, старайтесь через теги не прописувати правила, старайтеся усюди прописати класи.
- пункти у картках приліпити до лівої сторони картки, вони не мають бути по центру                                                                       - ul там де пункти в картках краще робити не через маржин-ботом до кожного пункту, а через дісплей флекс на увесь список, бо тоді там маржин з пункту йде маленький, а потім за ним маржин усього списку. Якщо зробите флексом ul, пропишите геп 16 пікселів, тоді просто там же пропишете маржин ботом 32 пікселя і буде вам відступ від списку до кнопки.                                                                                                                                                                                                 - text-align: left; у .pricing-list li ul li ні на що не впливає окрім одного елемента списку чомусь. Краще приберіть.
- padding: 0;
  background: none;
  border: none;
  box-shadow: none у .pricing-list li ul li, не розумію навіщо, краще розберіться чому у вас тінь з'являється на кожному пунктику без цих правил.                                                                                                                                                            Поки що все, потім будемо розбиратися, коли поправите попереднє (головне змініть одиниці з рем на пікселі та пропишіть кожному елементу на який ставите правило клас, бо це може і помилки викликати і для перевірки ну дуже важко. Вибачте що так багато ;\

Vegamond committed:
Add pricing section (HTML & CSS)

Vegamond committed:
Pricing Section fix

Vegamond committed:
Оновлено блок цін

kuroimera reviewed (changes requested):
-  з .section-pricing-text прибрати font-family: 'Sarabun', sans-serif; і font-weight: 400;
- можна в .section-pricing додати
    text-align: center; щоб по сто разів не переписувати. Видалити потім де дулблюється
- .section-pricing-title має фонт сайз бути 40 і летер спейсінг 0.01 ем. Також приберіть колір звідти
- в .section-pricing-text span всі стилі базові. Можете залишити тільки дисплей блок якщо це критично важливо.
- h3 class="pricing-item-title">One Session</h3 думаю тут можна зробити н6 заголовок
- а це для чого я не знайшла .tablet {
    display: none;
} можете не прибирати, просто зверніть увагу, може я не розумію просто.
- .pricing-item-title приберіть колір. там же летер спейсінг 0.01 ем у лайн хейт 1.4, маржин ботом 8 пікселів, а не 12. І продивіться ще, чи дійсно там треба обнуляти верхній маржин.
- .pricing-item-text там фонт-сайз 44, лайн хейт 1.2 і летер спейсінг 0.01 ем. Також приберіть колір
- .pricing-package-list додати верхній і нижній падінг по 8.
- приклад анімування кнопки, беріть стилі звідси тільки замініть на свої класи .join-button-header-menu {
  transition-property: background-color, border;
  transition-duration: 250ms;
  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
}

.join-button-header-menu:hover {
  background-color: rgba(194, 217, 252, 0.5);
  border: solid 1px rgba(194, 217, 252, 0.5);
}

.join-button-header-menu:focus {
  background-color: transparent;
}
- анімації у нас висять у окремому файлі. Туди все перенесіть
- у таблеті і в десктопі прибрати .section-pricing {
        padding-top: 40px;
        padding-bottom: 40px;
    }
 .section-pricing {
        padding-top: 112px;
        padding-bottom: 112px;
    }
- пропустила у .section-pricing-text {
    font-weight: 400; там жирність шрифта 600
-  .pricing-item-container {
            margin-bottom: 32px;
    } у таблеті прибрати, він створює подвійний маржин, ви вже до ціни вказали маржин ботом.
- у таблеті .section-pricing-list .section-pricing-item:last-child  justify-content: left; має бути
-  .pricing-item-text маржин бото 24 а не 34
- у таблеті  /* .pricing-item-container .pricing-item-text {
        margin-bottom: 24px;
    } оце прибрати
- @media only screen and (min-width: 1336px) {
    .section-pricing {
        padding-top: 112px;
        padding-bottom: 112px;
    }

    @media only screen and (min-width: 1336px) {
  .pricing-item-container {
    min-height: 148px;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: flex-start;
    text-align: center;
  } два рази продублювали медіазапит
- іконки не з'являться src/img/icons.svg з таким шляхом. Має бути так /img/icons.svg #name-of-the-icon (ім'яіконки берете з свдж спрайту)

Vegamond committed:
виправлення, додано коментарі

kuroimera reviewed (changes requested):
Зовсім трохи залишилось. Виправте це і будемо мерджити 
- .section-pricing {
    padding-top: 112px;
    padding-bottom: 112px;
    text-align: center;
}
видалити падінги
- повидаляти кольори з тексту, окрім посилань
- .pricing-item-container  додати маржин ботом 24
-  у мобайл додати .section-pricing-text span {
    margin-bottom: 48px;
  }
 у десктоп .section-pricing-text span {
    margin-bottom: 40px;
  }
- h3 class="pricing-item-title">One Session</h3  перероби у h6
- - пункти з галочками у ліво треба притиснути, зараз вони ближче до центру у вас
- прибрати з десктопу
  .section-pricing {
    padding-top: 112px;
    padding-bottom: 112px;
  }

Vegamond committed:
FIX

Виправлено помилки, видалено тестові кольори, доопрацьовано позіціонування третьої картки під таблет

kuroimera reviewed (changes requested):
- transform: scale(1.05); з анімацій повиирайте
- з секцій повидаляйте падінги .section-pricing {
    padding-top: 112px;
    padding-bottom: 112px;
    text-align: center;
}
- розібратися з картками (на планшеті 2, на мобайлі 5)
- пунктики з іконками у ліво мають бути

Vegamond committed:
FIX Pricing

kuroimera reviewed (changes requested):
*,
*::before,
*::after {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
.container {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 16px;
}
- ось це прибрати, воно ламає блок
- Перші два написи зроблені не так. Там не та товщина, не той розмір і т.д. і не той нижній маржин (там на планшеті мобайлі і десктопі вони різні, подивіться). (1 картинка)
- Третю картку в планшеті треба якось вертикально відцентрувати  (2 картинка)
- Якийсь джоін нау і хрестик.
- У .pricing-container неправильний gap: 40px, має бути 24 на десктопі і планшеті, в мобайлі 32.
на .priving-card не той нижній падінг на мобайлі. У вас 32 34 96, а має бути 32 24 110. - На планшеті і десктопі перевірте чи у вас це ж значення правильне. Має бути на планшеті 16 16 94 (на перших двух картках). На декстопі 32 32 110.
- У .pricing-card на мобілці gap 24, на планшеті і десктопі 32

Vegamond committed:
last fix
